### PR TITLE
Email services domain initiate and cancel verification changes

### PR DESCRIPTION
**Pull request by v-vprasannak**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by v-vprasannak** [comment]
> ## ⚠️ Suggestions
> ### Module: communication
> * Update version to `1.8.0b3` in setup.py
> * Set `azext.isPreview` to `true` in azext_communication/azext_metadata.json if not exists
> 
> #### Notes
> * Stable/preview tag is inherited from last release. If needed, please add `stable`/`preview` label to modify it.
> * Major/minor/patch/pre increment of version number is calculated by pull request code changes automatically. If needed, please add `major`/`minor`/`patch`/`pre` label to adjust it.
> * For more info about extension versioning, please refer to [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

1. setup.py file is referring to version.py file for version, in version.py file version is updated to '1.8.0b3'.
2. In azext_communication/azext_metadata.json file 'azext.isPreview' is set to 'true'.

File: src/communication/azext_communication/aaz/latest/communication/email/_list.py [hunk 1]
```diff
@@ -16,6 +16,10 @@
 )
 class List(AAZCommand):
     """List requests to list all resources in a subscription.
+
+    :example: Get all resources from a subscription/resource group
+        az communication email list --subscription SubscriptionId
+        az communication email list -g ResourceGroup
```
Comment (kairu-ms) [inline src/communication/azext_communication/aaz/latest/communication/email/_list.py:22]: actually the --subscription is optional. The command will use login subscription as default. BTW, we usually separate it into two examples like this:

```suggestion
    :example: Get all resources from a subscription
        az communication email list --subscription SubscriptionId

     :example: Get all resources from a resource group
        az communication email list -g ResourceGroup
```

Comment (v-vprasannak) [reply src/communication/azext_communication/aaz/latest/communication/email/_list.py]: separated examples as suggested.

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**